microsoft/testsuites/core/storage.py

```diff
@@ -581,18 +643,21 @@ def _hot_add_disk_serial(
         current_data_disk_count = node.capability.disk.data_disk_count
         log.debug(f"current_data_disk_count: {current_data_disk_count}")
 
-        # disks to be added to the vm
-        disks_to_add = max_data_disk_count - current_data_disk_count
-
         # get partition info before adding data disk
         partitions_before_adding_disk = lsblk.get_disks(force_run=True)
         added_disk_count = 0
         disks_added = []
-        for _ in range(disks_to_add):
+        # Assuming existing disks added sequentially from lun = 0 to
+        # (current_data_disk_count - 1)
+        free_luns = list(range(current_data_disk_count, max_data_disk_count))
+        if randomize_luns:
```

Comment:
Avoid to use random in the test case, it makes the test case behavior inconsistent.

Reply:
Used seed instead of just random